index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio-setup-mobile-first</title>
+  <link rel="stylesheet" href="styles.css">
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="navigation">
+      <p class="logo">Enoch</p>
+      <img src="/images/Union.png">
+    </div>
+  </header>
+  <section class="main">
+    <div class="intro">
+      <h2 class="para1">Hey There. I'm Enoch</h2>
+      <h3 class="para2">I'm a Software Developer</h2>
+      <p class="para3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+    </div>
+    <div class="icon">
+      <a href="https://github.com/TaiwoEnoch"><img src="./images/Icon -GitHub.png"></a>
+      <a href="#"><img src="./images/linkdln-icon.png"></a>
+      <a href="https://www.linkedin.com/in/taiwo-enoch"><img src="./images/Enabled.png"></a>
+      <a href="https://twitter.com/TaiwoEnoch4"><img src="./images/twitter-icon.png"></a>
+      <a href="https://medium.com/@taiwoenoch0"><img src="./images/Disabled.png"></a>
+    </div>
```

Comment:
- [ ] [REQUIRED] Nice work but the list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Please implement my suggestions ✔️